tests/dummy/app/routes/feed.js

```diff
@@ -0,0 +1,70 @@
+import Ember from 'ember';
+
+const {
+  Route,
+  Object: EmberObject,
+  computed: { sort }
+} = Ember;
+
+const { computed } = Ember;
+
+const FeedItem = EmberObject.extend({
+  title: null,
+  type: null,
+  createdAt: null,
+  displayDate: computed('createdAt', function () {
+    return `${this.get('createdAt').toDateString()}`;
+  })
+});
+
+const FeedItemsList = EmberObject.extend({
+  feedItems: null,
+  feedItemsSorting: [],
+  sortedFeedItems: sort('feedItems', 'feedItemsSorting'),
+});
+
+export default Route.extend({
+
+  queryParams: {
+    title: {
+      refreshModel: true,
+      replace: true,  // prevents an additional item from being added to browser history
+    },
+    type: {
+      refreshModel: true,
+      replace: true,
+    },
+    createdAt: {
+      refreshModel: false,  
+      replace: true,
+    }
+  },
+
+  model({ title, type }) {
+    const feedItems = type ?
```

Comment:
Should use `if` / `else` block level for these conditionals. Save the ternary operator for one liners.